addons/website_slides/controllers/main.py

```diff
@@ -794,6 +794,22 @@ def slide_archive(self, slide_id):
 
         return False
 
+    @http.route('/slides/category/archive', type='json', auth='user', website=True)
+    def slide_category_archive(self, category_id):
+        """ This route allows channel publishers to archive categories.
+        It has to be done in sudo mode since only website_publishers can write on slides in ACLs """
+        category = request.env['slide.slide'].browse(category_id)
+        slide_ids = category.slide_ids
+        if category.channel_id.can_publish:
+            for slide in slide_ids.sudo():
+                slide.category_id = False
+                category.channel_id.sudo()._resequence_slides(slide)
```

Comment:
What happens if you don't resequence for each slide but once after removing category_id of all slides ? Is it crashing ?

Reply:
If we just apply category_id = False it doesn't effect sequence of slides. Because category_id recompute based on sequence number. So at some point it will work on initial phase but as soon as we add new un-categorised slide it will spoil the sequence of slides and moved to another category which is not relevant to it. (ref: https://github.com/odoo/odoo/blob/master/addons/website_slides/models/slide_slide.py#L219)

So it requires to update sequence number just like when we are adding new un-categorised slide.